### ISSUE TITLE
Building fail.

### ISSUE THREAD

**Issue by Alleop5** [pineappleEA/pineapple-src#124]
When trying to build pineapple-src I get these errors.

https://pastebin.com/ukNyJPvT

I know this isn't yuzu but this seems relevant and if it's not it seems like a easy fix for someone who knows how to build yuzu.

**Comment by MGThePro** [comment]
build it from the source code in the windows release archive, not the source code on github.

And run `git submodule update --init --recursive` before the actual compilation step

**Comment by Alleop5** [comment]
I downloaded pineapple-src-EA-3628.tar.gz which is what I hope what you meant with the source code in the windows release archive. I then extracted and used git submodule update --init --recursive but I didn't git clone it so that didn't work with the error: fatal: not a git repository (or any of the parent directories): .git . Sorry if I understood you wrong.

**Comment by MGThePro** [comment]
No that's the source code from the repo. You need to download the ` Windows-Yuzu-EA-3628.zip ` and extract the `yuzu-windows-msvc-source-20230601-fd84145c7e.tar.xz` inside of that archive.

**Comment by Alleop5** [comment]
The source code is in a zip file where the application is for windows users. Makes sense. Anyway here are the next set of errors which don't even make sense. It's hilarious though.
 https://pastebin.com/hfQ9HTJD

**Comment by MGThePro** [comment]
Those first two errors are probably because you are trying to build inside the source code folder. Make sure to create and cd into a build folder first. For reference, see the [yuzu compilation guide](https://github.com/yuzu-emu/yuzu/wiki/Building-for-Linux)

**Comment by Alleop5** [comment]
Thank you for your help so far. Here are some more random errors: https://pastebin.com/xF8Kr2FP

**Comment by MGThePro** [comment]
Hmm, no idea.

It seems like it should also return the same error when building master/mainline, so maybe try building that first. If it does indeed have the same problem you might be able to get support from the yuzu devs.

**Comment by Alleop5** [comment]
Uninstalling llvm did the trick. Anyway now I'm having the issue called: "Yuzu not liking Wayland.". But it built and this is a yuzu issue so. Issue closed. Thank you for your help.

**Comment by MGThePro** [comment]
> Anyway now I'm having the issue called: "Yuzu not liking Wayland."

use the environment variable `QT_QPA_PLATFORM=xcb`. It'll be xwayland but at least it works

**Comment by Alleop5** [comment]
> use the environment variable QT_QPA_PLATFORM=xcb. It'll be xwayland but at least it works

That wasn't really an option as I wanted the most fps with as little glitches as possible. I figured it out though through the yuzu team which gave me this tip:

Try QT6. Works great for me.
-DENABLE_QT6=ON \

On second thought it might be using the bundles QT. Try setting
-DYUZU_USE_BUNDLED_QT=OFF

I used both options and it worked great. I can't currently test it though because I have a weak iGPU in the laptop I was compiling on. Which had 256 MB of VRAM which is obviously not enough for yuzu. And my main pc has a GTX 980 so I can't run Linux on that.
